packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -144,6 +166,17 @@ export async function getConsolidated({
         dateTo,
         conversionType,
       });
+
+      const duration = diffFromNow(startedAt);
+
```

Comment:
NABD, but I'd save empty lines to more important things. Getting the duration is totally related to its usage on `analytics`, to maybe keep it together? Would even consider just calling it from line 176 where we pass the `duration` param.